docs/ref/models/expressions.txt

```diff
@@ -660,6 +660,12 @@ should avoid them if possible.
     force you to acknowledge that you're not interpolating your SQL with user
     provided data.
 
+    To protect against SQL injection, you also must not quote placeholders in
+    the SQL string. For example, don't include quotes around the ``%s`` like
+    this::
+
+        RawSQL("select col from sometable where othercol = '%s'")  # bad!
+
```

Comment:
In the `extra()` docs below we have

```
Please read more about :ref:`SQL injection protection <sql-injection-protection>`.
```

Whilst here it's just an in-line link in the first paragraph. Make it an explicit one here too? It's important reading if you are using any of these features :)